tests/src/test/scala/system/basic/WskActionTests.scala

```diff
@@ -210,6 +253,32 @@ class WskActionTests extends TestHelpers with WskTestHelpers with JsHelpers with
     }
   }
 
+  it should "update an action with different language and check preserving params" in withAssetCleaner(wskprops) {
+    (wp, assetHelper) =>
+      val name = "updatedAction"
+
+      assetHelper.withCleaner(wsk.action, name, false) { (action, _) =>
+        wsk.action.create(
+          name,
+          Some(TestUtils.getTestActionFilename("hello.js")),
+          parameters = Map("name" -> testString.toJson)) //unused in the first function
+      }
+
+      val run1 = wsk.action.invoke(name, Map("payload" -> testString.toJson))
+      withActivation(wsk.activation, run1) { activation =>
+        activation.response.status shouldBe "success"
+        activation.logs.get.mkString(" ") should include(s"hello, $testString")
+      }
+
+      wsk.action.create(name, Some(TestUtils.getTestActionFilename("hello.py")), update = true)
+
+      val run2 = wsk.action.invoke(name)
+      withActivation(wsk.activation, run2) { activation =>
+        activation.response.status shouldBe "success"
+        activation.logs.get.mkString(" ") should include(s"Hello $testString")
+      }
```

Comment:
Do we need to run these actions? Do we even need a system test for it? Feels like an API level test "just" updating the action and then checking that "parameters" remain unchanged is sufficient here, WDYT?

Reply:
Perhaps we don't need to invoke it first time but we need it the second time in order to verify that the function can correctly consume the params even if it's a different runtime/language. This is more like end-to-end test.

Reply:
So here you are testing that the bound parameter is not lost? Did you check the actions api tests for a comparable test? If there’s one there do we need to add this? If so, should it also be out of the system suite.

Reply:
fair - I think the api test iirc (didn’t verify again) checks with the parameters with a get instead of invoke.

Reply:
`run2` could just be `run`? minor point.